controllers/cartController.js

```diff
@@ -25,6 +25,55 @@ const createCart = catchAsync(async (req, res) => {
   }
 });
 
+const getCart = catchAsync(async (req, res) => {
+  try {
+    const userId = req.user.id;
+
+    const carts = await cartService.getCart(userId);
+
+    res.status(201).json({ carts });
```

Comment:
```suggestion
    res.status(200).json({ carts });
```
`201` statusCode는 레코드 생성을 의미합니다!